docs/evaluation/simulate.rst

```diff
@@ -156,9 +245,16 @@ One way to include background is to sample it from a
 loaded into the dataset before running the simulation and, if not done
 before, the scale of the background scaling has to be set::
 
-  >>> data.set_background(read_pha('sherpa-test-data/sherpatest/9774_bg.pi'))
-  >>> data.backscal = 9.6e-06
-  >>> fake_pha(data, srcmdl, add_bkgs=True)
+   >>> bgdata = read_pha(data_dir + '9774_bg.pi')
+   >>> data.set_background(bgdata)
+   >>> data.backscal = 9.6e-06
+   >>> fake_pha(data, instmdl, add_bkgs=True)
```

Comment:
The code for `fake_pha` says:
```Python
    if add_bkgs is not None:
        warn("add_bkgs is no-longer used, as the model "
             "either contains a background term or the "
             "background datasets are used",
             category=DeprecationWarning)
```
So, I believe the background dataset will be used if present and `add_bkgs` should not be set all all (neither True nor False).
Or am I missing something obvious?